docs/source/quickstart_drilldown.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. _quickstart_drilldown:
+
+Quickstart options and troubleshooting
+============================================================
+
+..
+    Eventually, these H2s will probably all be separate articles. For now, I'm 
+    avoiding that so as not to create a bunch of super-skinny pages. 
+
+.. _quickstart_drilldown_install:
+
+Customize and troubleshoot MLflow installation
+---------------------------------------------------
+
+Python library options
+**********************
+
+Rather than the default MLflow library, you can install the following variations:
+
+.. list-table::
+   :widths: 10 20 70
+   :header-rows: 1
+
+   * - Name
+     - ``pip install`` command
+     - Description
+   * - mlflow-skinny
+     - ``pip install mlflow-skinny``
+     - Lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+   * - mlflow[extras]
```

Comment:
Do we want to include the 'install dev' version here as well? We end up telling users in reported issues to do this quite frequently to apply a merged-but-not-released hotfix to master quite often.